tft/script_train_fixed_params.py

```diff
@@ -41,6 +41,9 @@
 import numpy as np
 import pandas as pd
 import tensorflow.compat.v1 as tf
+tf.experimental.output_all_intermediates(
+    True
+)
```

Comment:
Could you explain why this is needed?

Reply:
I found the error
InvalidArgumentError: ... Connecting to invalid output 80 of source node ... which has 80 outputs. Try using tf.compat.v1.experimental.output_all_intermediates(True).

Reference: https://stackoverflow.com/questions/67856392/node-training-adam-gradients-gradients-conv5-block3-3-bn-cond-grad-statelessif

And I've tried with mine, it's work.